x-pack/plugins/observability/public/pages/alerts/styles.scss

```diff
@@ -0,0 +1,12 @@
+$fullscreenFlyoutTop: 72px;
+.kbnBody.euiBody--headerIsFixed.euiDataGrid__restrictBody {
```

Comment:
Again, makes me a little nervous how deeply we're nesting into EUI classes, **but** I can't think of a better solution right now 🤔